comments/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from .models import Comment
+
+
+class CommentSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Comment
+        fields = ["text", "post", "user", "timestamp"]
```

Comment:
1. As we are going to have a list of comments shown below the post, we don't need to serialize the `post` attribute for each entry. We can rather have `post_id` field
2. For the `user` field: [Nested relationships in DRF](https://www.django-rest-framework.org/api-guide/relations/#nested-relationships)